src/clims/plugins/demo/dnaseq/workflows/sequence.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+
+# TODO: Should we have a namespace from which plugin developers should import
```

Comment:
Sounds like a good idea to make it easier for developers to know that they are getting "the right stuff".